tools/audit-pre-flight.sh

```
@@ -1,76 +1,150 @@
-REMOTE=$(git rev-parse @{u}) # Remote commit hash
-
-if [ "$LOCAL" != "$REMOTE" ]; then
-    echo -e "${RED}[WARNING] Local HEAD differs from Origin!${NC}"
-    echo -e "${YELLOW}Please run 'git pull' or sync Brain Artifacts first.${NC}"
-    # Do not exit 1 here to allow offline work flexibility,
-    # but the agent will be clearly warned.
+# 2. GIT DRIFT CHECK
+echo -e "\n${YELLOW}Step 2: Checking Version Control Sync...${NC}"
+git -C "$ROOT_DIR" fetch origin > /dev/null 2>&1 || true
+LOCAL=$(git -C "$ROOT_DIR" rev-parse @)
+REMOTE=$(git -C "$ROOT_DIR" rev-parse @{u} 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)